src/SALib/analyze/delta.py

```diff
@@ -15,12 +15,15 @@ def analyze(
     conf_level: float = 0.95,
     print_to_console: bool = False,
     seed: int = None,
+    Nr: int = None,

```

Comment:
I propose to follow SciPy's naming here (that would be handy also if we do move towards using `scipy.stats.bootstrap`)

```suggestion
    n_resamples: int = None,

```

Reply:
I think I need someone to explain to me conceptually why we need two separate arguments to do with resampling.

If it is indeed necessary to have both, `n_resamples` is too close to the current `num_resamples`, but I'd like to avoid capitalization in arguments where possible, so agree `Nr` has to be changed...

Reply:
Oh yes I got confused myself as well. `n_resamples` is not a good name here. Sorry for the confusion. I am not sure about naming here, maybe `n_subsamples`?

Reply:
If you plan to use scipy.stats.bootstrap in the future, imho it would make sense to adopt the argument names `num_resamples` -> `n_resamples`

I would propose to name the size of the resample `Nr` -> `s_resamples`

Reply:
We have a backward compatibility concern here, we sadly cannot rename `num_resamples -> n_resamples` without a deprecation cycle.

`s_resamples`, I am concerned this would be understood as `sobol_resamples`. Since we would reduce Y, why not just `y_resamples` (or `Y_resamples` but I am not a fan of the casing)?

Reply:
If you go for my proposal to remove sobol indices from this method, that would no longer be a problem.

Reply:
Please don't remove Sobol' indices from this method. We cannot do this like that for backward compatibility reasons. This is going to break people's code.

Reply:
I'm okay with `y_resamples`, seems like a good interim compromise. 

I'm also open to removing Sobol' (not in this PR, but in the future).

Reply:
On second thought, I think best to leave the Sobol' estimation in (but optional).

I prefer `y_resamples`.